#!/usr/bin/env python
PACKAGE = "compliant_trajectory_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("trans_x", double_t, 0, "Translational tolerance in x-axis", 0, 0, 1000000)
gen.add("trans_y", double_t, 0, "Translational tolerance in y-axis", 0, 0, 1000000)
gen.add("trans_z", double_t, 0, "Translational tolerance in z-axis", 0, 0, 1000000)
gen.add("rot_x", double_t, 0, "Rotational tolerance in x-axis", 0, 0, 1000000)
gen.add("rot_y", double_t, 0, "Rotational tolerance in y-axis", 0, 0, 1000000)
gen.add("rot_z", double_t, 0, "Rotational tolerance in z-axis", 0, 0, 1000000)

exit(gen.generate(PACKAGE, "compliant_trajectory_control", "Tolerance6DOF"))